nixos/modules/services/networking/wireguard.nix

```diff
@@ -98,6 +98,19 @@ let
         "main".'';
       };
 
+      fwmark = mkOption {
+        default = null;
+        example = 51820;
+        type = types.nullOr types.int;
+        description = ''
+          Mark packets with this value in fwmark.
```

Comment:
Which packets does it mark? Overlay or underlay?

Reply:
I don't know.